test/clojure_tictactoe/cli/output/instructions_printer_test.clj

```diff
@@ -0,0 +1,11 @@
+(ns clojure-tictactoe.cli.output.instructions-printer-test
+  (:require [clojure.test :refer :all]
+            [clojure.string :as string]
+            [clojure-tictactoe.helpers :as helper]
+            [clojure-tictactoe.cli.output.instructions-printer :refer :all]))
+
+(testing "Game start"
+  (testing "print-game-intro"
+    (deftest intro-game-message-test
+      (testing "Intro messages"
+       (is (string/includes? (with-out-str (print-game-intro)) "TicTacToe"))))))
```

Comment:
You can also test whether the output includes `instructions-printer/welcome-message` and `instructions-printer/instructions-message`. Then you don't have to update the assertion if for whatever reason those messages end up changing (e.g. if you decide to change it to `Tic-Tac-Toe` or `tic tac toe`).

Reply:
🆒